### ISSUE TITLE
Detect and fix usage of `sap.ui.core` namespace in XMLView definition

### ISSUE THREAD

**Issue by flovogt** [SAP/ui5-linter#1024]
I define a `sap/ui/core/mvc/XMLView` but instead of using the recommended `sap.ui.core.mvc` namespace to define the View, I use the old `sap.ui.core` namespace.

```xml
<core:View
  xmlns:mvc="sap.ui.core.mvc"
  xmlns:core="sap.ui.core"
  xmlns="sap.m"
  controllerName="ui5app.controller.Home">
    <Text text="Hello"/>
<core:View>
```

# Detection Goal
This scenario is detected and a finding is reported to use the `sap.ui.core.mvc` namespace.

# Autofix Goal
The view definition is adjusted to consume the `sap.ui.core.mvc` namespace. In case the `sap.ui.core` is no longer used after the migration, it should be removed.

```xml
<mvc:View
  xmlns:mvc="sap.ui.core.mvc"
  xmlns:core="sap.ui.core"
  xmlns="sap.m"
  controllerName="ui5app.controller.Home">
    <Text text="Hello"/>
<mvc:View>
```

**Comment by flovogt** [comment]
- Detection will be implemented with CPOUI5FOUNDATION-1329
- Autofix will be implemented with CPOUI5FOUNDATION-1330
